src/app/store/telemetry-search.js

```diff
@@ -1,37 +1,22 @@
 import { readable } from 'svelte/store';
 
 import FlexSearch from 'flexsearch'
-const url = 'https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes'
+// TODO: Make this dynamic based on prod vs local dev.
+const url = 'http://localhost:8000/api/v1/probes';
 
-const resultToProps = (info, id) => {
-    const { history } = info;
-    let description;
-    if (history.release) description = history.release[0].description;
-    else if (history.beta) description = history.beta[0].description;
-    else description = history.nightly[0].description;
-    description = description.split("\n").map(p => `<p>${p}</p>`).join('')
-    let firstRelease = info.history.nightly;
-    let firstReleaseName = 'nightly';
-    if (!firstRelease) {
-        firstRelease = info.history.beta;
-        firstReleaseName = 'beta';
-    }
-    firstRelease = firstRelease[firstRelease.length-1].versions.first;
-    const out = {
-        name: info.name, probeType: info.type, description, id, firstRelease, firstReleaseName
-    }
-    return out;
-}
 
 const telemetrySearch = readable({loaded: false}, async (set) => {
-    let data = await fetch(url).then(r=>r.json())
-    data = Object.values(data).map(resultToProps)
+    const resp = await fetch(url).then(r=>r.json())
+    // TODO: Change API to return array of probe objects?
```

Comment:
@hamilton What's your vote? Would array of objects vs one big object be more API user friendly?

Reply:
I think how we have it now is fine, and we can change down the line. This pattern of mapping the keys / spreading the values is pretty performant in practice.